examples/common/python/connectors/tcf_connector.toml

```diff
@@ -28,17 +28,17 @@ type = "Ethereum"
 
 [ethereum]
 #Direct registry contract file
-direct_registry_contract_file = "common/tcf_connector/ethereum/contracts/WorkerRegistryList.sol"
+direct_registry_contract_file = "examples/common/python/connectors/ethereum/contracts/WorkerRegistryList.sol"
 #Worker registry contract file
-worker_registry_contract_file = "common/tcf_connector/ethereum/contracts/WorkerRegistry.sol"
+worker_registry_contract_file = "examples/common/python/connectors/ethereum/contracts/WorkerRegistry.sol"
 #Deployed contract address of direct registry contract address.
 #Initial deploy the contract using eth_cli.py to get the this address
-direct_registry_contract_address = "0xB033aFeb6205Bcc322aa93a713db65511a447DF3"
+direct_registry_contract_address = "0x9Be28B132aeE1b2c5A1C50529a636cEd807842cd"
 #Deployed contract address of worker registry contract address.
-worker_registry_contract_address = "0xf652A128606Af90C5B80961E8e29aeE8f1299038"
+worker_registry_contract_address = "0xC5cC1624d450AAcb37B2bc859f7d8632133dB041"
 
 #Ethereum account details
-eth_account = "0xDcB780ea30C1041abfcCCE4233c275B9F313eAf4"
+eth_account = "0xb03874AE05dCe37B162dd234d794d0e22FCdEec5"
 #Ethereum account private key should be set in environment variable WALLET_PRIVATE_KEY
```

Comment:
Since you changed the ethereum account id, did you set the WALLET_PRIVATE_KEY in Dockerfile?

Reply:
I could see WALLET_PRIVATE_KEY being set in other PR#23